src/json/builder.cr

```diff
@@ -95,6 +95,27 @@ class JSON::Builder
     end
   end
 
+  # Writes a big integer.
+  def number(number : BigInt)
```

Comment:
these changes should be in `src/bin/json` otherwise a `require "json"` will include libgmp (or not compile since `BigInt` is not defined)

Reply:
`src/big/json`

Reply:
I've moved them into `src/big/json`